### PR TITLE
add: more example websites

### PR DESCRIPTION
**Pull request by pwnwriter**

### CODE REVIEW

File: EXAMPLES.md [hunk 1]
```diff
@@ -43,3 +43,5 @@
 | [146 Parks](https://146parks.blog/)                                | https://github.com/scouten/146parks.blog                 |
 | [films.mlcdf.fr](https://films.mlcdf.fr)                           | https://github.com/mlcdf/films                           |
 | [Mish Ushakov](https://mish.co)                                    |                                                          |
+| [castor](https://castorisdead.xyz)                                 | https://github.com/whoisYoges/website                    |
+| [pwnwriter](https://pwnwriter.xyz)                                 | https://github.com/pwnwriter/www
```
Comment (Keats) [inline EXAMPLES.md]: This gives a 404?

Comment (pwnwriter) [reply EXAMPLES.md]: This site is under dev... will be updated within one or two day.

Comment (pwnwriter) [reply EXAMPLES.md]: Hi, i've removed the 404 site and add new websites built with zola. Hope the pr will get accepted:)

**Comment by Keats** [comment]
Thanks!

**Pull request merged**